example/models/city/v1/city.proto

```diff
@@ -20,6 +20,14 @@ message Attractions {
       "numid"
     ]
   };
+  option (pgdb.v1.msg).indexes = {
+    name: "furrrs"
+    method: INDEX_METHOD_BTREE_GIN,
+    columns: [
+      "tenant_id",
+      "zoo_shop🌮fur"
```

Comment:
tacos are fun, but any reason to not use a period `.`?

Reply:
I like them for debugging because they are self highlighting.  I'll switch it over.